rfhub2/templates/twocolumn.html

```diff
@@ -7,7 +7,7 @@
             <li>
               <label class="tree-toggler nav-header"
                      title="file path: {{collection.path}}">{{collection.name}}</label>
-              <ul class="list-group tree">
+              <ul class="list-group tree" style="display: none;">
```

Comment:
we do not have styles defined inline in html templates. I would put it either to css file or to `doc.js` as a default state, next to the logic responsible for toggling.

Reply:
I'm sure You're right, but I'm clueless regarding frontend.

Reply:
it is correct, but should be added to `doc.css` which has all the custom style definitions, bootstrap.min.css is third party and we should not add anything to it.